Tools/jit/_optimizers.py

```diff
@@ -566,6 +578,8 @@ class OptimizerAArch64(Optimizer):  # pylint: disable = too-few-public-methods
         rf"\s*(?P<instruction>{'|'.join(_branch_patterns)})\s+(.+,\s+)*(?P<target>[\w.]+)"
     )
 
+    # https://developer.arm.com/documentation/ddi0406/b/Application-Level-Architecture/Instruction-Details/Alphabetical-list-of-instructions/BL--BLX--immediate-
+    _re_call = re.compile(r"\s*blx??\s+(?P<target>[\w.]+)")
```

Comment:
Did you mean for this to be non-greedy?
```suggestion
    _re_call = re.compile(r"\s*blx?\s+(?P<target>[\w.]+)")
```

Reply:
Oh yes good catch it's supposed to be optional.